src/app/components/client/ExposuresFilterExplainer.tsx

```diff
@@ -0,0 +1,122 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styles from "./ExposuresFilter.module.scss";
+import Image from "next/image";
+import ModalImage from "../../components/client/assets/modal-default-img.svg";
+import { OverlayTriggerAria } from "react-aria";
+import { OverlayTriggerState } from "react-stately";
+import { useL10n } from "../../hooks/l10n";
+import { ModalOverlay } from "./dialog/ModalOverlay";
+import { Dialog } from "./dialog/Dialog";
+import { Button } from "../server/Button";
+
+type ExposuresFilterExplainer = {
```

Comment:
I think the linter is failing because the type `ExposuresFilterExplainer` has the same name as the component name `ExposuresFilterExplainer` you are exporting here.